internal/provider/resource_auth0_client.go

```diff
@@ -610,7 +608,6 @@ func newClient() *schema.Resource {
 				Optional: true,
 				ValidateFunc: validation.All(
```

Comment:
Considering now we only have 1 validation, let's skip using the `validation.All` and simply replace it with `validation.IsURLWithHTTPS` 🙏🏻 

```suggestion
				ValidateFunc: validation.IsURLWithHTTPS
```

Reply:
Fair point, fixed it.